### PR TITLE
Fix query order

### LINKED ISSUE #2739: Query editor incorrect sequence of queries after creating 9 queries
### Version Information
Latest 

### Environment
MacOS

### What is the expected behaviour?
Queries should be created in a sequential order

### What is the current behaviour?
Queries are not created in a sequential order after creating 9 queries

### Screenshots or Screencast
https://user-images.githubusercontent.com/12490590/161537486-27c711d3-e930-4b90-9f01-1cc617404407.mov

### PR DESCRIPTION
**Pull request by tanmoyopenroot**
#### Checklist
- [x] Includes tests
- [ ] Update documentation

#### Changes proposed in this pull request:
Sort queries using `createdAt`

#### Screenshot
Screenshot from 2022-05-04 15-17-11

### CODE REVIEW

**Review by gsmithun4 — APPROVED** [review]
LGTM

File: server/src/services/data_queries.service.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class DataQueriesService {
 
     return await this.dataQueriesRepository.find({
       where: whereClause,
-      order: { name: 'ASC' },
+      order: { createdAt: 'ASC' },
```
Comment (gsmithun4) [inline server/src/services/data_queries.service.ts:34]: Will it looks good if its in descending order?
Latest query at top?
@Navaneeth-pk

Comment (Navaneeth-pk) [reply server/src/services/data_queries.service.ts:34]: Yes 👍

**Review by gsmithun4 — CHANGES_REQUESTED** [review]
shall we change order to descending?

**Comment by tanmoyopenroot** [comment]
> shall we change order to descending?

Sure @gsmithun4, Will add the changes tomorrow

**Comment by tanmoyopenroot** [comment]
> shall we change order to descending?

Done

File: server/src/services/data_queries.service.ts (hunk 1 above)
Comment (tanmoyopenroot) [reply server/src/services/data_queries.service.ts]: Done

**Comment by gsmithun4** [comment]
deployed
https://query-order.herokuapp.com/

**Review by gsmithun4 — APPROVED** [review]
LGTM

**Pull request merged**